app/src/main/res/layout/contactpatient_callpatient.xml

```diff
@@ -91,7 +91,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_marginHorizontal="@dimen/spacing_24"
-    android:layout_marginTop="@dimen/spacing_16"
+    android:layout_marginTop="@dimen/spacing_10"
```

Comment:
Looks like we didn't fix the order of the content as shown in the designs. Will you be making a new PR for it?

![Screenshot 2022-05-25 at 6.02.12 AM.png](https://graphite-user-uploaded-assets.s3.amazonaws.com/KqrtAphEKX3ow2MZW1lw/cce9fee2-4674-46f2-869b-bb962d94af7f/Screenshot%202022-05-25%20at%206.02.12%20AM.png)